tests/v2/libraryTest/library_test.go

```diff
@@ -8,17 +8,10 @@ import (
 	libraryUtils "github.com/devfile/library/tests/v2/utils/library"
 )
 
-// TestContent - structure used by a test to configure the tests to run
-type TestContent struct {
-	CommandTypes   []schema.CommandType
-	ComponentTypes []schema.ComponentType
-	FileName       string
-	EditContent    bool
-}
-
 func Test_ExecCommand(t *testing.T) {
 	testContent := commonUtils.TestContent{}
 	testContent.CommandTypes = []schema.CommandType{schema.ExecCommandType}
+	testContent.AddEvents = commonUtils.GetBinaryDecision()
```

Comment:
`AddEvents ` randomly set event values: https://github.com/devfile/api/blob/b08e48618403f7102592389e621531145bdfcbcc/test/v200/utils/common/events-utils.go#L35
looks like you cannot control to be only `Exec` or `Apply` or `composite` command.
Should not be under test for each particular type of command.

Reply:
ok, I see what you mean.  SetEventValues will create extra commands that will be outside the scope of the Exec testing, so I need to narrow this down to just the commands that Exec or composite Exec supports

Reply:
I think the easiest thing to do in this case is what you suggested; to remove events testing in these scenarios (otherwise I'd have to introduce new functions in the api utils).  I added new events test cases similar to what's being done in api tests, so that should ensure sufficient coverage.